tests/rsp_gen_minimal/Makefile.in

```diff
@@ -0,0 +1,3 @@
+TOP_FILE := $(TEST_DIR)/dut.v
+TOP_MODULE := dut
+SURELOG_FLAGS := --disable-feature=parametersubstitution
```

Comment:
Is this option needed? I thought that it may even not be supported in current Surelog.

Reply:
Thanks for noticing that, seems to be left over after copying initial test setup. Removed.